middleman-core/fixtures/sass-assets-path-app/config.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-set :sass_assets_paths, [
+config[:sass][:load_paths] = [
```

Comment:
This is a user-facing API change. Can we avoid this?

Reply:
Done.